src/main/app/controller/users/UserController.ts

```diff
@@ -1,12 +1,12 @@
 import {AuthedRequest} from '../../../types/AuthedRequest';
 import {Response} from 'express';
-
+import config from 'config';
 
 export class UserController {
 
   public async get(req: AuthedRequest, res: Response): Promise<void> {
-
-    res.render('users/index');
+    const userDashboard: string = config.get('services.idam.userDashboard');
+    res.redirect(userDashboard);
```

Comment:
If you want you can shorten this to one line: 

res.redirect(config.get('services.idam.userDashboard')); 

it should be bringing it across as a string based on the type its created as in the config file

Reply:
fixed